scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -321,75 +321,60 @@ public void onClick(ClickEvent clickEvent) {
      *                +-----------+
      * </pre>
      */
+    @SuppressWarnings("squid:S3776")
     private ToolStrip buildTools() {
         ToolStrip tools = new ToolStrip();
         tools.setHeight(50);
         tools.setWidth100();
         tools.setBackgroundImage("");
-        tools.setBackgroundColor("#fafafa");
+        tools.setBackgroundColor(logoStripBackgroundColor);
         tools.setBorder("0px");
 
         MenuItem submitMenuItem = new MenuItem("Submit job",
                                                SchedulerImages.instance.job_submit_16().getSafeUri().asString());
-        submitMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                SubmitWindow w = new SubmitWindow(SchedulerPage.this.controller);
-                w.show();
-            }
+        submitMenuItem.addClickHandler(event -> {
+            SubmitWindow w = new SubmitWindow();
+            w.show();
         });
         MenuItem flatSubmitMenuItem = new MenuItem("Submit command file",
                                                    SchedulerImages.instance.script_16().getSafeUri().asString());
-        flatSubmitMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                FlatSubmitWindow w = new FlatSubmitWindow(SchedulerPage.this.controller);
-                w.show();
-            }
+        flatSubmitMenuItem.addClickHandler(event -> {
+            FlatSubmitWindow w = new FlatSubmitWindow(SchedulerPage.this.controller);
+            w.show();
         });
 
         MenuItem settingsMenuItem = new MenuItem("Settings", Images.instance.settings_16().getSafeUri().asString());
-        settingsMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                SchedulerPage.this.settingsWindow.show();
-            }
-        });
+        settingsMenuItem.addClickHandler(event -> SchedulerPage.this.settingsWindow.show());
 
         MenuItem credMenuItem = new MenuItem("Create credentials", Images.instance.key_16().getSafeUri().asString());
-        credMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                CredentialsWindow credentialsWindow = new CredentialsWindow();
-                credentialsWindow.show();
-            }
+        credMenuItem.addClickHandler(event -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)